src/sentry/api/endpoints/project_plugin_details.py

```diff
@@ -87,6 +90,30 @@ def post(self, request: Request, project, plugin_id) -> Response:
         if not plugin.can_disable:
             return Response({"detail": ERR_ALWAYS_ENABLED}, status=400)
 
+        if integrations.exists(plugin_id):
+            for feature in integrations.get(plugin_id).features:
+                feature_flag_name = "organizations:integrations-%s" % feature.value
+                if not features.has(feature_flag_name, project.organization):
```

Comment:
@oioki some of the mapped feature flags don't even exist as a plan feature. They are purely integration features. I think we need logic to ignore any of those non-existent feature flags.

Reply:
Would this work?
```suggestion
                if feature_flag_name in settings.SENTRY_FEATURES and not features.has(feature_flag_name, project.organization):
```